turbinia/workers/__init__.py

```diff
@@ -750,6 +761,7 @@ def run_wrapper(self, evidence):
     with filelock.FileLock(config.LOCK_FILE):
       log.info('Starting Task {0:s} {1:s}'.format(self.name, self.id))
       original_result_id = None
+      turbinia_worker_tasks_total.inc()
```

Comment:
I wonder if we should add a "queued" metric that gets set further up in this method? As of now, this metric will not increment until after the task is actually started, but Tasks can get queued on a worker for quite a while as other Tasks run.

Reply:
Added queued tasks metrics